src/System.Net.Security/src/System/Net/Security/TlsCipherSuiteData.OpenSslName.Lookup.Linux.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
remove

Reply:
I'm not sure what the "remove" is for.  If it's the file, OK.  If it's the license header, no... it should still be around.

Reply:
I meant the whole file, initially it was doing a bit more but after couple of iterations it ended up being redundant...